examples/dht_node.rs

```diff
@@ -192,9 +191,12 @@ fn add_ping_sender(base_selector: IoFuture<()>, server_obj: &Server) -> IoFuture
 }
 fn add_nat_sender(base_selector: IoFuture<()>, server_obj: &Server, pk: PublicKey) -> IoFuture<()> {
     // 3 seconds for NatPingRequest
-    let nat_wakeups = Interval::new(Duration::from_secs(3));
+    let interval = Duration::from_secs(3);
+    let nat_wakeups = Interval::new(Instant::now() + interval, interval);
     let server_obj_c = server_obj.clone();
-    let nat_sender = nat_wakeups.for_each(move |()| {
+    let nat_sender = nat_wakeups
+        .map_err(|_err| Error::new(ErrorKind::Other, "NatPing timer error"))
```

Comment:
Maybe we shouldn't swallow these errors? They can be useful if this error ever happens.